src/components/layout/RightSectionWallet.vue

```diff
@@ -21,25 +21,50 @@
         {{ address }}
       </div>
       <div class="not-bootstrap-row">
-        <AeAmount :amount="balance" />
+        <Dropdown
+          v-if="hasContractV2Address"
+          :options="tokenBalancesOptions"
+          :method="updateTokenBalance"
+          :selected="selectedTokenBalance.token"
+        >
+          <template #diplayValue>
+            <AeAmount
+              class="not-bootstrap-row"
+              :amount="selectedTokenBalance.balance"
+              :token="selectedTokenBalance.token"
+            />
+          </template>
+          <template slot-scope="{ option }">
```

Comment:
`slot-scope` is deprecated, let's use a new syntax instead
https://vuejs.org/v2/guide/components-slots.html#Scoped-Slots-with-the-slot-scope-Attribute